src/cascadia/ShellExtension/pch.cpp

```diff
@@ -2,3 +2,5 @@
 // Licensed under the MIT license.
 
 #include "pch.h"
+
+#pragma comment(lib, "User32.lib")

```

Comment:
Can we add this in the `.vcxproj` file instead? None of our other projects use the pragma style of including libs, and I wouldn't want someone to get confused in the future. You should be able to add it with 

```xml

  <ItemDefinitionGroup>
    <Link>
      <AdditionalDependencies>User32.lib;%(AdditionalDependencies)</AdditionalDependencies>
    </Link>
  </ItemDefinitionGroup>
```

Reply:
alright